app/scripts/views/sign_up.js

```diff
@@ -61,8 +63,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
 
       this._bouncedEmail = this.ephemeralMessages.get('bouncedEmail');
-
-      return FormView.prototype.beforeRender.call(this);
+      getNow(['passwordcheck'], function (PasswordCheck) {
+        self.passwordcheck = new PasswordCheck();
```

Comment:
so passwordcheck is the function name defined in the Password-Strength-Checker repo, and passwordCheck is a wrapper for using that in the signup.js page. Changed both to better reflect meaning and difference.